Remotion/Web/Core/UI/Controls/WebTreeNodeCollection.cs

```diff
@@ -23,6 +25,18 @@ namespace Remotion.Web.UI.Controls
   /// <summary> A collection of <see cref="WebTreeNode"/> objects. </summary>
   public class WebTreeNodeCollection : ControlItemCollection
   {
+    /// <summary>
+    /// Groups the top level <see cref="WebTreeNode"/>s by their <see cref="WebTreeNode.Category"/>
+    /// </summary>
+    /// <param name="webTreeNodes">The nodes of the <see cref="WebTreeView"/></param>
+    /// <returns></returns>
+    public static IReadOnlyList<WebTreeNode> GroupByCategory (WebTreeNodeCollection webTreeNodes)
```

Comment:
There's no need to use `webTreeNodes` as the parameter name, but it mostly pops up because it was `nodes` in the earlier commits. Both work, I guess.